less/index.less

```diff
@@ -78,20 +206,21 @@ table {
 thead {
 	color: #13abc4;
 }
+
 .container {
-    width: 1080px;
-    margin: 0 auto;
-		padding-bottom: 20px;
+	width: 1080px;
+	margin: 0 auto;
+	padding-bottom: 20px;
 }
 
 .navigation {
-	  display: flex;
-		flex-direction: row;
-		padding-top: 15px;
-		padding-bottom: 35px;
-		justify-content: space-between;
-		align-items: center;
-		font-size: 2rem;
+	display: flex;
+	flex-direction: row;
+	padding-top: 15px;
+	padding-bottom: 35px;
+	justify-content: space-between;
+	align-items: center;
+	font-size: 2rem;
 }
 
 nav a {
```

Comment:
Your `nav a` should be nested inside your nav. There are a few nesting misses on this page so see if you can resolve those